assets/src/design-system/components/input/index.js

```diff
@@ -65,7 +66,6 @@ const InputContainer = styled.div(
     justify-content: space-between;
     height: 36px;
     padding: 4px 12px;
-    background-color: ${theme.colors.bg.primary};
```

Comment:
@BrittanyIRL We are only using transparent background inputs in the editor and it seemed at the first glance when checking that it's the same in the Dashboard as well. Let me know if you see any issues with removing the background-color, I might have missed something.

Reply:
go for it! This was just an honest oversight building in storybook.